content/en/webhooks/webhook-approval.md

```diff
@@ -74,7 +109,7 @@ webhooks:
 {{< /prism >}}
 
 
-## How to trigger a webhook
+### How to trigger a webhook
 
 You can trigger a webhook from the following areas:
 
```

Comment:
- A step within a canary strategy
- The `redirectTrafficAfter` and `shutDownOldVersionAfter` section of a blue/green strategy